tests/tests_twobody/test_orbit.py

```diff
@@ -641,82 +641,115 @@ def test_plane_is_set_in_horizons():
 
 
 @pytest.mark.parametrize(
-    "attractor,angular_velocity,expected_a,expected_period",
+    "attractor,expected_a,expected_period",
     [
-        (
-            Earth,
-            (2 * np.pi / 23.9345) * u.rad / u.hour,
-            42_164_205 * u.m,
-            23.9345 * u.hour,
-        ),
-        (
-            Mars,
-            (2 * np.pi / 24.6228) * u.rad / u.hour,
-            20_427_595 * u.m,
-            24.6228 * u.hour,
-        ),
+        (Earth, Earth.R + 35786 * u.km, Earth.rotational_period,),
+        (Mars, Mars.R + 17031 * u.km, Mars.rotational_period,),
     ],
 )
-def test_geostationary_creation_from_angular_velocity(
-    attractor, angular_velocity, expected_a, expected_period
-):
-    ss = Orbit.geostationary(attractor=attractor, angular_velocity=angular_velocity)
-    assert_quantity_allclose(ss.a, expected_a, rtol=1.0e-7)
-    assert_quantity_allclose(ss.period, expected_period, rtol=1.0e-7)
+def test_stationary_orbit(attractor, expected_a, expected_period):
+    ss = Orbit.stationary(attractor=attractor)
+    assert_quantity_allclose(ss.a, expected_a, rtol=1.0e-4)  # SEMIMAJOR AXIS ASSERT
```

Comment:
I think we can do away with these comments (`# SEMIMAJOR AXIS ASSERT` and others)